docs/static-analysis.md

```diff
@@ -101,14 +101,29 @@ The plugin for IntelliJ can be found [here](https://plugins.jetbrains.com/idea/p
 To introduce code that violates PMD rules, use `@SuppressWarnings("PMD.RuleName")` annotation at the narrowest possible scope. PMD also provides several other methods of suppressing rule violations, which can be found in the [documentation here](https://pmd.github.io/pmd-5.8.1/usage/suppressing.html).
 The suppression should be as specific as possible, and the reason for violating the rule should be explained.
 
-### FindBugs
+### SpotBugs
 
-[FindBugs](http://findbugs.sourceforge.net) analyses Java source code for potential bugs at bytecode level, thus able to find potential bugs that PMD cannot find.
+[SpotBugs](https://spotbugs.github.io/) analyses Java source code for potential bugs at bytecode level, thus able to find potential bugs that PMD cannot find.
 In Gradle build, the rules are configured by specifying the classes in the `visitors` variable.
-The plugin for Eclipse can be found [here](http://findbugs.cs.umd.edu/eclipse/).
-The plugin for IntelliJ can be found [here](https://plugins.jetbrains.com/idea/plugin/3847-findbugs-idea).
 
-> You can also [configure all the static analysis tools automatically](#intellij-automatic-setup) for IntelliJ IDEA.
```

Comment:
It doesn't require setting up because the detectors that are enabled are the defaults that are shipped with the plugin.

That can be changed easily of course, but it seems we are not configuring it for Eclipse and IntelliJ previously either, so it was left unchanged.

Reply:
Let's just leave it unchanged until we have "officially" included more rules for FindBugs/SpotsBugs in our CI.